app/code/Magento/QuoteGraphQl/Model/Resolver/CartAddresses.php

```diff
@@ -43,6 +43,11 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
 
         $cart = $value['model'];
 
-        return $this->addressDataProvider->getCartAddresses($cart);
+        return $this->addressDataProvider->getCartAddresses($cart, $this->includeShippingMethods($info));
+    }
+
+    private function includeShippingMethods(ResolveInfo $info): bool
```

Comment:
If we declare separate resolver for `available_shipping_methods` in a scheme we don't need this class